### PR TITLE
[Create survey] Do NOT display message error when user input min/max …

### PR DESCRIPTION
**Pull request by nampx-0860**
Summary: Do NOT display message error when user input min/max content of linear scale question with 256 character

Step to reproduce: 
1. Open the create survey screen
2. Choose linear scale question
3. Input min content = 256 character
4. Observe the screen

Actual result: Do NOT display message error when user input min/max content of linear scale question with 256 character

Expected result: Display message error: "Please enter no more than 255 characters."

### CODE REVIEW

**Comment by nampx-0860** [comment]
minmaxcontent
https://edu-redmine.sun-asterisk.vn/issues/12666

**Comment by nampx-0860** [comment]
image
https://edu-redmine.sun-asterisk.vn/issues/12458

**Comment by nampx-0860** [comment]
image
https://edu-redmine.sun-asterisk.vn/issues/12458

File: resources/lang/vn/validation.php [hunk 1]
```diff
@@ -132,7 +132,7 @@
         'digits' => 'Xin vui lòng chỉ nhập các chữ số.',
         'creditcard' => 'Số thẻ tín dụng không hợp lệ.',
         'equalTo' => 'Vui lòng nhập giá trị tương tự một lần nữa.',
-        'maxlength' => 'Hãy nhập không lớn hơn {0} ký tự.',
+        'maxlength' => 'Vui lòng nhập tối đa {0} ký tự.',
```
Comment (tranducquoc) [inline resources/lang/vn/validation.php]: Chỉ được phép nhập tối đa {0} ký tự.

**Pull request merged**